git-integration/README.md

```diff
@@ -0,0 +1,8 @@
+# Roadmap for Git Integration
+
+Many notebook users utilize git and GitHub to manage versions of
+their notebooks and share them with others. Sometimes this process
+can be quite difficult as creating diffs of the notebook format
+comes with a lot of challenges. We are hoping to address this
+issue and create tightly-coupled git integration in the nteract
+desktop application.
```

Comment:
Noted! I'll probably add more technical context to these initiatives sometime this week and will ping you to review the one on git integration.